spec/helpers/external_links_helper_spec.rb

```diff
@@ -141,6 +155,16 @@
         )
       end
     end
+
+    context 'with content-publisher' do
+      it 'uses the specific label' do
+        expect(
+          edit_label_for(publishing_app: 'content-publisher')
+        ).to eq(
+          'Edit in Content publisher'
```

Comment:
Can we use the locales in the test? - `I18n.t('...')`
```

Reply:
I don't see the advantage looking at all the other tests in the file. Also we are testing the 
translations kook up correctly.

Reply:
So the reasoning was that if we do make content changes, content designers can just make them and not have to worry about the tests.

Reply:
Done now